pkgs/by-name/kc/kconf/package.nix

```diff
@@ -14,14 +17,25 @@ buildGoModule rec {
   vendorHash = "sha256-REguLiYlcC2Q6ao2oMl92/cznW+E8MO2UGhQKRXZ1vQ=";
 
   ldflags = [
-      "-s" "-w" "-X github.com/particledecay/kconf/build.Version=${version}"
+    "-s"
+    "-w"
+    "-X github.com/particledecay/kconf/build.Version=${version}"
   ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
```

Comment:
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Reply:
what are you suggesting exactly ? I don't really see the purpose of this 😅